### ISSUE TITLE
Segfault on Linux when calling OS.open_midi_inputs()

### ISSUE THREAD

**Issue by kallistisoft** [godotengine/godot#51169]
### Godot version

v3.3.1.rc1.official

### System information

ArchLinux, GLES2, NVIDIA GeForce GTX 1060/PCIe/SSE2, nvidia-dkms 465.31-1

### Issue description

Calling OS.open_midi_inputs() from GDScript causes a segmentation fault. 

```
Godot Engine v3.3.1.rc1.official - https://godotengine.org
OpenGL ES 2.0 Renderer: NVIDIA GeForce GTX 1060/PCIe/SSE2
OpenGL ES Batching: ON
 
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /usr/lib/libc.so.6(+0x3cda0) [0x7f6504d7bda0] (??:0)
-- END OF BACKTRACE --
[1]    32186 abort (core dumped)  ./executable.x86_64
```

### Steps to reproduce

Attach the following script to the root node of a scene

```
extends Spatial

func _ready():
        OS.open_midi_inputs()
```

### Minimal reproduction project

[MIDIDemo.tar.gz](https://github.com/godotengine/godot/files/6916273/MIDIDemo.tar.gz)

**Comment by qarmin** [comment]
Duplicate of #46183
Will be fixed in Godot 3.4

**Comment by kallistisoft** [comment]
Thank you
